charts/datadog/v2.0.14/Chart.yaml

```diff
@@ -1,14 +1,14 @@
 apiVersion: v1
 name: datadog
-version: 1.33.0
-appVersion: 6.13.0
-description: DataDog Agent
+version: 2.0.14
+appVersion: "7"
+description: Datadog Agent
 keywords:
 - monitoring
 - alerting
 - metric
 home: https://www.datadoghq.com
-icon: file://../dd_logo_70x75.png
+icon: https://datadog-live.imgix.net/img/dd_logo_70x75.png
```

Comment:
u may keep using the existing icon `icon: file://../dd_logo_70x75.png` so the air-gapped environment user can view it properly

Reply:
done